homeassistant/helpers/llm.py

```diff
@@ -687,6 +687,14 @@ def _get_exposed_entities(
         if include_state:
             info["state"] = state.state
 
+            # Convert timestamp device_class states from UTC to local time
+            device_class = state.attributes.get("device_class")
+            if device_class == "timestamp" and state.state:
```

Comment:
i think this can fit on one line also
```suggestion
            if state.attributes.get("device_class") == "timestamp" and state.state:
```